styles.css

```diff
@@ -126,6 +126,13 @@ article div {
     background-color: #A91725;
 }
 
+.clear {
+    display: flex;
+    text-align: center;
+    align-items: center;
+
```

Comment:
- remove this line break
- on the wireframe, the clear search button is red not green
- we need to take a look at how this appears on mobile - clear search button should probably drop below search bar to leave maximum space for typing